build-aux/logfile-uploader.py

```diff
@@ -26,96 +25,111 @@
 COUCH_URL = "https://couchdb-vm2.apache.org/ci_errorlogs"
 TARFILE = "couchlog.tar.gz"
 
+
 def _tojson(req):
     """Support requests v0.x as well as 1.x+"""
-    if requests.__version__[0] == '0':
```

Comment:
I wonder if it would be better to `--skip-string-normalization` and avoid all the conversions from single to double quotes? I don't personally find their [opinions on preferring double quotes](https://github.com/ambv/black#strings) entirely compelling.